src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let total = 0;
+
+  if (days >= 7) {
+    total = days * 40 - 50;
+  } else if (days >= 3) {
+    total = days * 40 - 20;
+  } else {
+    total = days * 40;
```

Comment:
Check point 1 in checklist. For now it's hard to understand what is `40`, `20`, `50`. Let's create descriptive variables for these values